src/WIP-Draft - Governance Process.md

```diff
@@ -38,9 +38,13 @@ providers and consumers - in direct connection with one another, reducing interm
 all users to act in the marketplace knowing that they have *supreme transparency and fair price
 discovery*.
 
-We are determined to make WindingTree Protocol the **canonical marketplace** for the trading of 
+We are determined to make WindingTree Protocol the **recognized marketplace** for the trading of 
 *real-world services* by applying continuous innovation, encouraging intellectual discourse and
-expanding the WindingTree Protocol ecosystem - inviting the brightest minds to join the community. 
+expanding the WindingTree Protocol ecosystem - inviting the brightest minds to join the community.
+
+Some examples of marketplaces that can be transformed with the WindingTree protocol are Online Travel Agencies
+(e.g. Booking.com), Global Distribution Systems (e.g. Amadeus) or even e-commerce platforms (e.g. Amazon).
```

Comment:
I think with respect to "e-commerce platforms", I would argue that if something like this were included, it would be expansionary on the scope of "real-world services", ie. Amazon as an e-commerce platform is more about selling products / goods, and not services (with the exclusion of AWS).